pyclesperanto_prototype/_tier9/_statistics_of_background_and_labelled_pixels.py

```diff
@@ -6,7 +6,7 @@
 from .._tier0 import pull_zyx
 
 @plugin_function(output_creator=create_none)
-def statistics_of_background_and_labelled_pixels(input : Image, labelmap : Image):
+def statistics_of_background_and_labelled_pixels(input : Image, labelmap : Image, extra_properties=[]):
```

Comment:
This is fine although note that skimage allows measuring regionprops *without* pixel intensities, and this is useful. Not sure whether you want to support that use case...

Reply:
also fwiw the American spelling is `labeled`. I don't have a strong opinion on US vs UK spelling but Clesperanto should pick one and stick to it. Generally I see most people using US in code. (e.g. neighbors vs neighbours).

Reply:
I see, because it's connected_components_labeling with one L. Pitty. I created as issue for this #66. I assume there are many more methods that could have better names (e.g. more similar to scikit-image, or imagej or matlab), so we can collect some ideas like this one. Thanks again!

Reply:
I have an idea: How about German method names? Instead of "connected_component_labeling" it would then be "kennzeichnung_verbundener_komponenten". That might reduce typos and mistakes on my side dramatically 🤣